xarray/tests/test_dask.py

```diff
@@ -4,6 +4,8 @@
 import pickle
 import numpy as np
 import pandas as pd
+import pytest
+import mock
```

Comment:
On Python 3, use `from unittest import mock` to avoid adding the new dependency.